src/uct/api/uct_def.h

```diff
@@ -490,22 +653,18 @@ typedef void (*uct_ep_disconnect_cb_t)(uct_ep_h ep, void *arg);
  * Communication progress routines should not be called from this callback.
  * It is allowed to call other UCT communication routines from this callback.
  *
- * @param [in]  arg        User defined argument for this callback.
- * @param [in]  dev_name   Device name. This routine may fill the user's private
- *                         data according to the given device name.
- *                         The device name that is passed to this routine,
- *                         corresponds to the dev_name field inside
- *                         @ref uct_tl_resource_desc_t as returned from
- *                         @ref uct_md_query_tl_resources.
- * @param [out] priv_data  User's private data to be passed to the remote side.
+ * @param [in]  arg          User defined argument for this callback.
+ * @param [in]  pack_handle  Handle for the the private data packing.
+ * @param [out] priv_data    User's private data to be passed to the remote side.
  *
  * @return Negative value indicates an error according to @ref ucs_status_t.
  *         On success, a non-negative value indicates actual number of
  *         bytes written to the @a priv_data buffer.
  */
-typedef ssize_t (*uct_sockaddr_priv_pack_callback_t)(void *arg,
-                                                     const char *dev_name,
-                                                     void *priv_data);
+typedef ssize_t
+(*uct_sockaddr_priv_pack_callback_t)(void *arg,
+                                     uct_sockaddr_priv_data_pack_cb_handle_t
```

Comment:
const

Reply:
rename this callback to uct_cm_ep_priv_data_pack_cb_t
even though this callback is used for non-cm (old) flow for now, let's have the name for the new API since old sockadd flow will be deprecated